lib/ansible/modules/cloud/ovh/ovh_monthly_billing.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Francois Lallart (@fraff)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = '''
+---
+module: ovh_monthly_billing
+author: Francois Lallart (@fraff)
+version_added: "2.10"
+short_description: Manage OVH monthly billing
+description:
+    - Enable monthly billing on OVH cloud intances (be aware OVH does not allow to disable it).
+requirements: [ "ovh" ]
+options:
+    project_id:
+        required: true
+        type: str
+        description:
+            - ID of the project, get it with https://api.ovh.com/console/#/cloud/project#GET
+    instance_id:
+        required: true
+        type: str
+        description:
+            - ID of the instance, get it with https://api.ovh.com/console/#/cloud/project/%7BserviceName%7D/instance#GET
+'''
+
+EXAMPLES = '''
+# basic
+  - ovh_monthly_billing: project_id=0c727a20aa144485b70c44dee9123b46 instance_id=8fa89ad2-8f08-4220-9fa4-9695ea23e948
+
+# more complex
+  - os_server_facts:
+      cloud: myProjectName
+      region_name: myRegionName
+      server: myServerName
+    # force run even in check_mode
+    check_mode: no
+
+  - name: ovh_monthly_billing
+    ovh_monthly_billing:
+      project_id: "{{ openstack_servers.0.tenant_id }}"
+      instance_id: "{{ openstack_servers.0.id }}"
+'''
+
+RETURN = '''
+'''
+
+import os
+import sys
+
+try:
+    import ovh
+    import ovh.exceptions
+    from ovh.exceptions import APIError
+    HAS_OVH = True
+except ImportError:
+    HAS_OVH = False
+
+from ansible.module_utils.basic import AnsibleModule
```

Comment:
```suggestion
from ansible.module_utils.basic import AnsibleModule, missing_required_lib
```

Reply:
I can't modify that, it won't run on my python2.7 setup